invoices_client.go

```diff
@@ -228,13 +230,43 @@ func (s *invoicesClient) AddHoldInvoice(ctx context.Context,
 	rpcCtx, cancel := context.WithTimeout(ctx, s.timeout)
 	defer cancel()
 
+	var routeHints []*lnrpc.RouteHint
+
+	for _, rh := range in.RouteHints {
```

Comment:
Why not `routeHints, err := marshallRouteHints(in.RouteHints)`?

Reply:
Oh indeed I also didn't see that!

Reply:
Oh, did not see this comment!
Much better, changing